### ISSUE TITLE
dayDblClick and eventDblClick

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#638]
Originally reported on Google Code with ID 367
```
I'd like to see the Dbl Click events exposed in the same way that dayClick
and eventClick are.

Don
```

Reported by `DonaldWBabcock` on 2010-02-25 22:45:21

**Comment by DominikDitoIvosevic** [comment]
"because it opens the
doors to people wanting many more of the more obscure events (like touch events), and
where do you draw the line"

Is it really that hard to implement such events that you have to "draw the line" on "obscure" functions like doubleclick?
All I had to do to manually implement dblclick was to do 

`
				dblclick: function (seg, ev) {
				    return view.trigger('eventDoubleClick', this, seg.event, ev);`

**Comment by arshaw** [comment]
ya, i've changed my stance on this one. especially since we are using event delegation for this stuff now, and as a result there is no performance issue with this, i can see this being a good idea now

**Comment by koybe** [comment]
when the cell is covered with a event, while the z-index is bigger than the fc-bg,so the click event won't take effect,how can i do?

**Comment by arshaw** [comment]
internally, click event's aren't triggered via a `click` event, they use a "HitDragListener" (see the src code), which is more complicated, thus the difficulty of this feature.

**Comment by koybe** [comment]
this is my code, i attach the event throug dayRender callback:
![image](https://cloud.githubusercontent.com/assets/1536470/16791381/f42db606-48f0-11e6-8445-2562456af131.png)

@arshaw

**Comment by arshaw** [comment]
right, but with that technique, you will run into the z-index troubled you mention. the HitDragListener sidesteps all that

**Comment by koybe** [comment]
what is the HitDragListener , i don't know it very clearly
could you give a sample code for me ? thanks a lot ^_^

**Comment by arshaw** [comment]
it's not a matter of sample code. it's a matter of taking the time to build this feature the right way, which is more involved and not a quick fix

**Comment by koybe** [comment]
awesome! 
does it in your plan and when can I try it？

**Comment by ashilay** [comment]
Hi guys, do you plan to extend clicking experience with the event described here?

**Comment by RipkensLar** [comment]
Any news on this?

**Comment by ItsNoHax** [comment]
Any updates on this? Requested feature
